sheets/api.py

```diff
@@ -1102,8 +1103,8 @@ def get_assignments_to_create_and_remove(
             if already_redeemed_assignments:
                 log.error(
                     "Cannot remove ProductCouponAssignments that are already redeemed - "
-                    "The following assignments will not be removed: %s"
-                    % str(list(already_redeemed_assignments))
+                    "The following assignments will not be removed: %s",
+                    str(list(already_redeemed_assignments)),
```

Comment:
`str` here is redundant since the `%s` will convert the python object to a string using `str` first: https://docs.python.org/3/library/stdtypes.html#printf-style-string-formatting